src/main/java/org/sonarlint/intellij/core/ServerUpdateTask.java

```diff
@@ -101,8 +103,18 @@ public void run(@NotNull ProgressIndicator indicator) {
   }
 
   private void updateModule(ConnectedSonarLintEngine engine, ServerConfiguration serverConfiguration, String key) {
-    engine.updateModule(serverConfiguration, key);
-    log.log("Module '" + key + "' in server binding '" + server.getName() + "' updated", LogOutput.Level.INFO);
+    try {
+      engine.updateModule(serverConfiguration, key);
+      log.log("Module '" + key + "' in server binding '" + server.getName() + "' updated", LogOutput.Level.INFO);
+    } catch (Exception e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1166)